frontend/public/components/dashboard/dashboards-page/cluster-dashboard/details-card.tsx

```diff
@@ -49,6 +61,10 @@ const ClusterVersion: React.FC<ClusterVersionProps> = ({ cv }) => {
       name: 'version',
     }) && window.SERVER_FLAGS.branding !== 'dedicated';
 
+  if (!(canUpgrade && hasAvailableUpdates(clusterVersion))) {
```

Comment:
I don't think you need this check since the existing `default` case will cover what is needed once line 85 is updated.